docs/ui.md

```diff
@@ -60,17 +60,20 @@ If mutiple players in a group are being streamed to then this is shown by a numb
 
 There are four ways to group players
 
-1.    **MANUAL SYNC.** As shown in the [Player List](#player-list) section players can be linked via the chain icon in the player list. Only like player types that support synced playback are allowed to be joined. These players will then play in sync. It should be noted that when players are grouped in this way only the main player holds the queue. For example, player A has an existing queue and players B and C are joined to player A. If player A is turned off, players B and C will also stop. This type of group is created when the HA action [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) is used.
-2.    **SYNC GROUP.** Create a group player via the settings. The native player types shown (i.e. not the Universal Group option) support perfect sync (this is a requirement to be shown here). In addition to playing in sync the group will also keep playing regardless of which player(s) in the group are powered off. So the group player holds the queue rather than any individual player. 
+1.    **MANUAL SYNC.** As shown in the [Player List](#player-list) section players can be linked via the chain icon in the player list. Only like player types that support synced playback are allowed to be joined. These players will then play in sync. It should be noted that when players are grouped in this way only the main player holds the queue. For example, player A has an existing queue and players B and C are joined to player A. If player A is turned off the queue will be lost and players B and C will stop. This type of group is created when the HA action [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) is used.
+2.    **SYNC GROUP.** Create a group player via the settings. The native player types shown (i.e. not the Universal Group option) support perfect sync (this is a requirement to be shown in the list). In addition to playing in sync the group will also keep the queue regardless of which player(s) in the group are powered off. So the group player holds the queue rather than any individual player. Note however that if the group leader is lost then playback will stop but the queue can be restarted.
```

Comment:
```suggestion
2.    **SYNC GROUP.** Create a (permanent) group player via the settings. In addition to playing in sync the group will also keep the queue on the group player itself, instead of on one of its child players.
```

There is too much info - we should try to make compact sentences that stick to the essence.
Nice FAQ's thingies can be stuffed somewhere else.

Maybe dedicate a page to grouping ?

Reply:
Text slimmed down. Majority moved to new groups page